.pre-commit-config.yaml

```diff
@@ -1,5 +1,11 @@
 repos:
 - repo: https://github.com/psf/black
-  rev: 19.3b0
+  rev: 19.10b0
   hooks:
   - id: black
+
+- repo: https://github.com/asottile/blacken-docs
```

Comment:
I'm not confident in pre-commit to enforce its configuration. It's optional for a developer to enable/install, so I see it as a convenience measure and not part of any standard workflow. If this change is to apply to every project, is it intended to be installed by anyone developing the project? Should these checks be included as part of the tests (the way the black style is enforced)?

Reply:
> If this change is to apply to every project, is it intended to be installed by anyone developing the project?

It should be up to each project to decide whether they'd want this. I think pre-commit is useful for developers, but for open source projects, should remain optional for those who aren't keen on it.

> Should these checks be included as part of the tests (the way the black style is enforced)?

Yes, I think pre-commit is a handy place to gather together lint checks, and include them on the CI, even if some developers don't want to run it locally.

So if blacken-docs isn't something we'd want to enforce, let's remove it from here.